marketplace/netlify-build/src/pubnub-client.js

```diff
@@ -0,0 +1,86 @@
+import PubNub from 'pubnub';
+
+function isObject(value) {
+  var type = typeof value;
```

Comment:
We've got `var` and `!=` in this function. I also don't understand why we allow functions. I also believe this code will allow an array.

I'd do:

```js
const isObject = val => typeof val === 'object' && val !== null && !Array.isArray(val)
```

Reply:
Yep, I can change these. Originally I copied them from `lodash` so I wouldn't need to import the whole package just to get some very minor functionality.